mappings/net/minecraft/block/WallTorchBlock.mapping

```diff
@@ -3,5 +3,6 @@ CLASS net/minecraft/unmapped/C_pqbshjyj net/minecraft/block/WallTorchBlock
 	FIELD f_kducrbwy BOUNDING_SHAPES Ljava/util/Map;
 	FIELD f_nqqcptkl SHAPE_OFFSET F
 	FIELD f_touxqlyd CODEC Lcom/mojang/serialization/MapCodec;
+	METHOD m_asezpade isSideSolidFullSquare (Lnet/minecraft/unmapped/C_eemzphbi;Lnet/minecraft/unmapped/C_hynzadkk;Lnet/minecraft/unmapped/C_xpuuihxf;)Z
```

Comment:
```suggestion
	METHOD m_asezpade canPlaceAt (Lnet/minecraft/unmapped/C_eemzphbi;Lnet/minecraft/unmapped/C_hynzadkk;Lnet/minecraft/unmapped/C_xpuuihxf;)Z
```
this can be narrowed down a bit, since it's in specifically `WallTorchBlock`